packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -718,7 +718,11 @@ class _DraggableScrollableSheetState extends State<DraggableScrollableSheet> {
 
   @override
   void dispose() {
-    widget.controller?._detach(disposeExtent: true);
+    if (widget.controller != null) {
```

Comment:
Can you revert it o make more readable:

`if (widget.controller == null) {`

Reply:
@polina-c thanks a lot for the review!
Added the fix.